### ISSUE TITLE
[request] haptic feedback on click

### ISSUE THREAD

**Issue by Niggolo** [rileytestut/DeltaCore#11]
Hello man!
It is possible to add an option to disable the vibration on every click I do on the screen?

**Comment by Niggolo** [comment]
Any news about the update? I’m using delta directly from ipa because I can’t use alt store, so I can’t see any update

**Comment by blueingsh** [comment]
yes there is, go to delta then click the settings icon then scroll down a bit until you see haptic feedback, you can turn on/off the options
